apps/admin-gui/src/app/shared/components/members-candidates-list/members-candidates-list.component.ts

```diff
@@ -38,14 +47,36 @@ export class MembersCandidatesListComponent implements OnChanges, AfterViewInit
     if (!!this.dataSource) {
       this.dataSource.sort = this.sort;
 
-      this.dataSource.sortingDataAccessor = (richMember, property) => {
+      this.dataSource.sortingDataAccessor = (memberCandidate, property) => {
         switch (property) {
+          case 'status':
+            return memberCandidate.member ? memberCandidate.member.status : '';
           case 'fullName':
-            return parseFullName(richMember.richUser);
+            if (memberCandidate.richUser) {
+              return memberCandidate.richUser.firstName.toLowerCase() + ' ' +
```

Comment:
There is actually a possibility that the firstName and lastName can be empty. In that case, it causes an error when calling the toLowerCase() function. Is there any reason why should you transform the names to lower case?

Reply:
well because Adam Adamek is on top of the table and aadam adamek is on last page of the table(yes a lot of names are in lower case)